From b53e8a51dea9b6ba76734d77def41bb15b577cdb Mon Sep 17 00:00:00 2001
From: Andrew Annex <ama6fy@virginia.edu>
Date: Mon, 15 Nov 2021 12:29:20 -0500
Subject: [PATCH] patch for cf

---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 114fde9..376932a 100644
--- a/setup.py
+++ b/setup.py
@@ -130,12 +130,10 @@ readme.close()
 # https://setuptools.pypa.io/en/latest/deprecated/distutils/extending.html?highlight=cmdclass#integrating-new-commands
 
 setup(
-    distclass=SpiceyPyBinaryDistribution,
     packages=find_packages('src'),
     package_dir={"": "src"},
     package_data={"spiceypy": ["utils/*.so", "utils/*.dylib", "utils/*.dll"]},
     include_package_data=True,
-    cmdclass=cmdclass, 
 )
 
 
-- 
2.33.0

