From 07eee55bfe7f4699359ce75cdb0145cd8da891ad Mon Sep 17 00:00:00 2001
From: AndrewAnnex <ama6fy@virginia.edu>
Date: Fri, 10 Jan 2020 10:40:13 -0500
Subject: [PATCH] get rid of setup_requires

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 283b8b0..02ca075 100644
--- a/setup.py
+++ b/setup.py
@@ -135,7 +135,6 @@ setup(
     include_package_data=True,
     zip_safe=False,
     package_data={"spiceypy": ["utils/*.so", "utils/*.dll"]},
-    setup_requires=DEPENDENCIES,
     install_requires=DEPENDENCIES,
     requires=REQUIRES,
     tests_require=TEST_DEPENDENCIES,
-- 
2.24.1

