From 735435b79dcd45a62775bbfbdfd35d174ed33e66 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Juan=20Luis=20Cano=20Rodr=C3=ADguez?= <juanlu001@gmail.com>
Date: Sun, 24 Jun 2018 19:00:36 +0200
Subject: [PATCH 2/2] Get rid of setup_requires

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git setup.py setup.py
index 12d6694..d44f2ff 100644
--- setup.py
+++ setup.py
@@ -88,7 +88,6 @@ setup(
     include_package_data=True,
     zip_safe=False,
     package_data={'spiceypy': ['utils/*.so', "utils/*.dll"]},
-    setup_requires=DEPENDENCIES,
     install_requires=DEPENDENCIES,
     requires=REQUIRES,
     tests_require=TEST_DEPENDENCIES,
-- 
2.7.4

